Changelog.md

```
@@ -1,4 +1,57 @@
+
+**Server**
+* `EntityAssociationAttribute` should now be used instead of `AssociationAttribute` (which is obsolete on .NET)
+    * While there is code for backwards compatibility please update all code to use
+
+**Client**
+
+* XML Support
+Added a new class  `OpenRiaServices.Client.DomainClients.XmlHttpDomainClientFactory` that can be used to enable text based XML communication with the server.
+
+  Sample usage:
+ 
```

markdownlint-cli2 (0.22.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)